src/Components/News/Detail/NewsDetailLayout.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect,useState } from 'react';
+import { useParams } from 'react-router';
+import NewsTitle from './NewsTittle';
+import { Box } from '@material-ui/core';
+import { getNewById } from '../../../Services/getNewById';
+
+const NewsDetailLayout = ({prop}) => {
+
+    const [newData,setNewData] = useState({})
+
+    const {id} = useParams()
+
+    useEffect(()=>{
+        const loadNewData = async () => {
+            const {data} = await getNewById()
+            setNewData(data)
+        }
+        loadNewData()
+    },[id])
+    
+    return ( 
+
+        <div>
+            <NewsTitle
+                title={prop.title}
+            />
+            <img alt="newImg" src={newData.img}></img>
+            <Box>
+                {newData.body}
+            </Box>
+        
+        </div> 
+
+    );
+}
+ 
```

Comment:
please, some general feedback:
- remove unnecessary spaces
- all the times that you can put the code in only one line.

Same feedback on src/Components/News/Detail/NewsTittle.js file.

Reply:
Sory about that, already fix it.